packages/zowe-explorer/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed failure to refresh token value after user logs in to authentication. [#2638](https://github.com/zowe/vscode-extension-for-zowe/issues/2638)
 - Fixed order of spool files reverses when the Job is expanded and collapsed. [#2644](https://github.com/zowe/vscode-extension-for-zowe/pull/2644)
 - Fixed local filtering of jobs to work with SMFID (exec-member field). [#2651](https://github.com/zowe/vscode-extension-for-zowe/pull/2651)
+- Fixed issue seen after renaming an MVS or USS file or folder. [#2656](https://github.com/zowe/vscode-extension-for-zowe/issues/2656)
```

Comment:
Should we be more specific here? Something like "Fixed tree item labels failing to update after renaming..."

Reply:
Hey Timothy, thanks added your suggestion in Prepare 2.14.0 Release [PR](https://github.com/zowe/vscode-extension-for-zowe/pull/2661/files)